src/foundry/rollTerms/poolTerm.d.ts

```diff
@@ -30,7 +30,7 @@ declare class PoolTerm extends RollTerm {
   modifiers: PoolTerm.TermData['modifiers'];
 
   /** The elements of a Dice Pool must be Roll objects or numbers */
-  rolls: (PoolTerm.TermData['rolls'] | number)[];
+  rolls: PoolTerm.TermData['rolls'];
```

Comment:
Why was that changed?

Reply:
Because the type is only `Roll[]` and not a union of rolls and numbers. Asked Atropos about it